_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,187 @@
+  example, the `MyPojo` type in `ValueState<Tuple2<Integer, MyPojo>>` or
+  `ListState<Either<Integer, MyPojo>>`, which is a POJO, is allowed to evolve
+  its schema.
+
+  For a full overview of the list of data types that now support schema
+  evolution as well as their evolution specifications and limitations, please
+  refer to the [State Schema
+  Evolution](https://ci.apache.org/projects/flink/flink-docs-master/dev/stream/state/schema_evolution.html)
+  documentation page.
+
+* **Java-serialization free savepoints**: Previously, when writing the metadata
+  for state in savepoints, the state serializers were also Java-serialized as
+  part of the metadata. This caused several problems, especially for state
+  schema evolution.
+
+  Back in Flink 1.7.0, we introduced new serialization compatibility
+  abstractions for the `TypeSerializer` interface to remove this behaviour. In
+  Flink 1.8.0, all of Flink's built-in `TypeSerializer`s have been upgraded to
+  use the new serialization compatibility abstractions, and therefore will no
+  longer be Java-serialized into savepoints.
+
+  This greatly improves Flink's savepoint interoperability and schema
+  evolvability. For users that are using custom implemented `TypeSerializer`s
+  for state serialization, this also means that they are no longer tied to the
+  serializer's implementation, as long as they have also upgraded their
+  `TypeSerializer` to the new serialization compatibility abstractions. For
+  instructions on how to do that, please refer to the [Custom State
+  Serialization](https://ci.apache.org/projects/flink/flink-docs-master/dev/stream/state/custom_serialization.html)
+  documentation page, as well as the class-level Javadocs of the
+  `TypeSerializer` class.
+
+* **Continuous cleanup of old state based on TTL
+  ([FLINK-7811](https://issues.apache.org/jira/browse/FLINK-7811))**: We
+  introduced TTL (time-to-live) for Keyed state in Flink 1.6
+  ([FLINK-9510](https://issues.apache.org/jira/browse/FLINK-9510)). This
+  feature enabled cleanup and making keyed state entries inaccessible after a
+  defined timeout. In addition state would now also be cleaned up when
+  writing a savepoint/checkpoint.
+
+  Flink 1.8 introduces continous cleanup of old entries for both the RocksDB
+  state backend
+  ([FLINK-10471](https://issues.apache.org/jira/browse/FLINK-10471)) and the heap
+  state backend
+  ([FLINK-10473](https://issues.apache.org/jira/browse/FLINK-10473)). This means
+  that old entries (according to the TTL setting) are continously being cleanup
+  up.
+
+* **SQL pattern detection with user-defined functions and aggregations**: The
+  support of the MATCH_RECOGNIZE clause has been extended by multiple features.
+  User-defined function are now possible which allows for custom logic during
+  pattern detection
+  ([FLINK-10597](https://issues.apache.org/jira/browse/FLINK-10597)).
+  Aggregations allow for more complex CEP definitions
```

Comment:
"while adding aggregations allows for more complex CEP definitions ... such as the following..."